src/core_codemods/docs/pixee_python_sql-parameterization.md

```diff
@@ -1,6 +1,6 @@
 This codemod refactors SQL statements to be parameterized, rather than built by hand.
 
-Without parameterization, developers must remember to escape string inputs using the rules for that column type and database. This usually results in bugs -- and sometimes vulnerability. Although it's not clear if this code is exploitable today, this change will make the code more robust in case the conditions which prevent exploitation today ever go away.
+Without parameterization, developers must remember to escape string inputs using the rules for that column type and database. This usually results in bugs -- and sometimes vulnerabilities. Although we can't tell for sure if your code is actually exploitable, this change will make the code more robust in case the conditions which prevent exploitation today ever go away.
```

Comment:
You might want to also update the Java version of SQL parameterization. This was lifted straight from there.